scalafix/output/src/main/scala/migrate/ExplicitResultTypesBug.scala

```diff
@@ -7,7 +7,7 @@ object ExplicitResultTypesBug {
 
```

Comment:
Why is this test called `Bug` if it is not failing?

Reply:
It was called like that in scalafix .. I guess there were adding bugs and then fixing them. I can change the name